.azure-pipelines-steps.yml

```diff
@@ -18,19 +18,18 @@ steps:
       versionSpec: '2.7'
     displayName: 'Use Python 2.7'
 
-  # Run yarn to install dependencies and build
-  - script: node scripts/remove-postinstall
-    displayName: 'Remove postinstall script'
-
   - task: CacheBeta@0
     inputs:
       key: yarn | $(Agent.OS) | yarn.lock
       path: $(YARN_CACHE_FOLDER)
```

Comment:
@willsmythe hiya! this line doesn't work with yarn@2. What's the syntax for running a command here? I wanna run `yarn config get cacheFolder`

Reply:
Hey @SimenB. Long time. Try this ...

1. Drop the `YARN_CACHE_FOLDER` variable declaration in `.azure-pipelines.yml`

2. Add a new step before the `CacheBeta@0` step that stores the cache folder path in a variable (`yarnCache.folder`), which can then be accessed later
    ```yaml
    bash: echo "##vso[task.setvariable variable=folder;isOutput=true]$(yarn config get cacheFolder)"
    name: yarnCache
    ```

3. Change the `path` input on the cache step to: `$(yarnCache.folder)`

Reply:
thanks! gave it a try in https://github.com/facebook/jest/pull/10188/commits/489e6178c09d142066ae56ef0d70da7f519d3ece

Reply:
that seems to have worked on mac and linux, but not windows

![image](https://user-images.githubusercontent.com/1404810/85567559-42e24800-b631-11ea-81f0-bb25a12a2fee.png)

Not sure if this is yarn or azure pipeline? I see double `C\:\\C\:\\` - is that the erorr or jsut weird output?

Reply:
I believe this is on us - Yarn is returning a portable path on Windows (ie `/c:/Users/...`) instead of the native ones. I'll look into that.

Reply:
Will be fixed by https://github.com/yarnpkg/berry/pull/1514, which will land in master in a few hours. Starting from there, you'll be able to run `yarn set version from sources` to install a fresh build (or you can run `yarn set version from sources --branch 1514` now).

Reply:
it worked! thanks both 👍